pyhealth/datasets/base_ehr_dataset.py

```diff
@@ -403,6 +411,7 @@ def set_task(
             self.patients.items(), desc=f"Generating samples for {task_name}"
         ):
             samples.extend(task_fn(patient))
+        samples.append(self.code_vocs)
         sample_dataset = SampleEHRDataset(
```

Comment:
Though it works, writing codes in this way is hard for later maintainers to understand. Please improve this part.